Library/Homebrew/extend/git_repository.rb

```diff
@@ -4,111 +4,111 @@
 require "utils/git"
 require "utils/popen"
 
-# Extensions to {Pathname} for querying Git repository information.
+# Helper methods for querying Git repository information.
 # @see Utils::Git
 # @api private
-module GitRepositoryExtension
+module GitRepository
```

Comment:
```suggestion
module Git
```
or 
```suggestion
module GitPath
```

Reply:
I went with `GitRepoPath` based on feedback from others.

Reply:
@dduugg not huge on the `Repo` shortening, personally but I can live with it if you love it.